src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
```

Comment:
as I see, it looks like this
```
<>
  <div>
  </div>
<>
```
in such cases, you don't need these `<>` fragments